packages/framework/view-interfaces/src/mountableView.ts

```diff
@@ -49,3 +49,12 @@ export interface IFluidMountableView extends IProvideFluidMountableView {
 	 */
 	unmount(): void;
 }
+
+/**
+ * TODO: determine what is needed here
+ */
+export interface IFluidMountableViewEntryPoint {
+	getDefaultDataObject(): Promise<FluidObject>;
+	getDefaultView(): Promise<IFluidMountableView>;
```

Comment:
With suggested change above to distinguish from the mountable version this becomes
```suggestion
	getDefaultView(): Promise<FluidObject>;
```